src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -109,34 +91,6 @@ function makeMoveButtonsEnabledStates({ topBoundaryMet, bottomBoundaryMet, right
   };
 }
 
-// If any of the image dimensions are greater than the max specified,
-// resize it down to that dimension while keeping the aspect ratio
-// intact
-function resizeIfAboveMaxDimension(img, mimeType, maxDimension) {
```

Comment:
I removed this because after testing on my Android phones some more, it seems like the data url serialization was the big performance issue, not image size. iPhones are faster than Android and shouldn't have much of an issue either. The resizing here was making it essentially impossible to work with large files, since they would get resized down to where you couldn't reasonable zoom in on them.